src/jasonwynn10/Nicknames/Main.php

```diff
@@ -15,10 +15,14 @@ class Main extends PluginBase implements Listener {
 
 	/** @var Config $nicknameDB */
 	protected $nicknameDB;
-
+	
+    public $economy;
 	public function onEnable() {
 		$this->getServer()->getPluginManager()->registerEvents($this, $this);
 		$this->nicknameDB = new Config($this->getDataFolder()."Nicknames.json", Config::JSON);
+		if ($this->getServer()->getPluginManager()->getPlugin("EconomyAPI")) {
```

Comment:
This would be better as a config check rather than a test to see if the plugin is available